client/components/listingsView.jsx

```diff
@@ -76,25 +79,25 @@ const PrevBtn = styled.div`
   stroke-width: 3;
   display: ${props => props.counter === 0? 'none' : ''};
   position: relative;
-  bottom: 160px;
-  left: 20px;
+  bottom: 180px;
+  left: 17px;
   z-index: 10;
   font-size: 30px;
-  color: white;
+  color: rgb(230,230,230);
   cursor: pointer;
 `;
 
 const NextBtn = styled.div`
   align-self: flex-end;
```

Comment:
Why use align-self: flex-end and then use bottom to push the div up 180px? Why not use align-self: center, and remove the bottom property?